cornell-dti/office-hours
# Added modal for are you sure you want to leave queue

### Summary 


I added a modal presentation for when a student is about to leave the queue.

A modal pops up that asks if a user wants to leave the queue whenever the user clicks the remove button on a question. I didn't implement the functions of the two buttons on the bottom yet, so as far as functionality goes, **the buttons do not do anything yet**. I will get this done very soon in the future.


- New modal presentation




### Test Plan 


- Add a question as a student
- Try removing the question
- See if the modal shows, and if you are able to exit out of the modal with the "X" button
### Notes 



### Breaking Changes 

None
